### DISCUSSION TITLE
Roasting with Background Roast

### DISCUSSION THREAD

**Discussion by stephenpate** [artisan-roaster-scope/artisan#1125] [Q&A]
So, I just asked this question via Bugs, but it may just be me. 

I was roasting today with a background roast profile loaded. I came in to the TP a bit hot and overshot my projected TP by about 15 degrees. I was going to ride at 0 power until about 30 sec prior to hitting my target and then increase power so I don't stall. When I tried to increase power, Artisan software would not allow me to increase power. It would go up to 30 and then back to 0. Once my live roast hit the projected roast, it restored functionality, and I was able to adjust power again, but by this point I had stalled and was 30-40 degrees behind until drop. 

Any ideas? Its probably something I did! Thanks!

**Comment by bigl00z3** [comment]
I just saw that you also posted here.

I paste below the reply I already posted in [Issues](https://github.com/artisan-roaster-scope/artisan/issues/1124):

> This is not an artisan issue or a bug. It is the way the PID is reacting based on your configuration.
> 
> You need to first have the good P, I and D values for your roaster and then you also need to configure everything around in Artisan to work well. Hoping to use the PID function without a proper configuration will almost for sure not work properly. If you are too hot and the background curve is lower, Artisan PID will remove power to reach the background.
> 
> You can read about PID and Artisan on:
> 
> - [the Artisan blog](https://artisan-roasterscope.blogspot.com/2016/11/pid-control.html)
> - In [this discussion list](https://github.com/artisan-roaster-scope/artisan/discussions?discussions_q=PID) by searching for `PID`
> 
> Nico
> 

PID control is great when well configured, but until it is, the results are generally far worse than roasting manually.

And also, if you want others to be able to use, you should provide more details about your installation. Which hardware, what type of connections and anything specific. 
Screenshots of your Artisan profile or pictures of your installation are very useful to describe what you mean and also attach the roast `.alog` file (the profile) and your artisan settings (`.aset` file which you can export via `/Help/Save Settings... `) to your post.
Note: to attach the files, do it on the GitHub page (not by email) and rename the files with .txt extension at the end of the name for it to upload properly.

**Comment by Dgocof** [comment]
Is this the first time using this back round profile? Did it react different previously? I am new to Artisan  also and am noticing  that the preheat  temperature will  set up  first part of the roast. I do different batch sizes in my roaster and I'm trying to dial in the preheat temps for each. This is something that I noticed in my previous system that I have been using for 20 +years.  The momentum created by your preheat temp and what the burners are doing when you drop can create a lot of momentum in the beginning of the roast. I would suggest lowering your drop temp a bit and playing with your PID settings more as stated by Nico.  The shape of your back round profile in the beginning could maybe be telling your burners something opposite of what you want. Look at that also.
